### PR TITLE
Bug/markup cleanup + bug fix

### PR DESCRIPTION
**Pull request by pris54**
This PR has markup fixes and tweaks so that the code validates.
There is a bug fix re' a repeated title.
It includes linting fixes too.

### CODE REVIEW

File: admin/views/sign-in.ejs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
         <p class="lede" id="lead-paragraph">Sign-in to access the Check Development Service</p>
 
-        <form action="/sign-in" method="post" aria-label="Sign-in below" autocomplete="off" autocapitalize="none">
+        <form action="/sign-in" method="post" aria-label="Sign-in below" autocomplete="off">
```
Comment (jon-shipley) [inline admin/views/sign-in.ejs]: Please either leave autocapitalise here or move it to the input fields.  It's needed on iOS tablets.

File: admin/views/sign-in.ejs [hunk 2]
```diff
@@ -20,7 +20,7 @@
                        aria-required="true"
                        aria-label="Enter your user name."
                        spellcheck="false"
-                       autocorrect="off">
+                       autocomplete="off">
```
Comment (jon-shipley) [inline admin/views/sign-in.ejs]: `autocorrect` is, I think, needed for iOS.

File: admin/views/sign-in.ejs (hunk 1 above)
Comment (pris54) [reply admin/views/sign-in.ejs]: @jon-shipley 'autocapitalize' comes up as invalid for element <form> - it throws a validation error (not warning). I found out that it's a "nonstandard attribute used by iOS Safari Mobile" - If you need it I'll put it back with a comment in both the markup and the README file.

File: admin/views/sign-in.ejs (hunk 2 above)
Comment (pris54) [reply admin/views/sign-in.ejs]: @jon-shipley same as above.

**Pull request closed without merging**